MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmTestPointCheckLib.c

```diff
@@ -250,19 +218,15 @@ TestPointSmmReadyToBootSmmPageProtection (
   EFI_STATUS  Status;
   BOOLEAN     Result;
 
-  if ((mFeatureImplemented[TEST_POINT_INDEX_BYTE6_SMM]
-    & TEST_POINT_BYTE6_SMM_READY_TO_BOOT_SMM_PAGE_LEVEL_PROTECTION) == 0) {
-    return EFI_SUCCESS;
-  }
-
   DEBUG ((DEBUG_INFO, "======== TestPointSmmReadyToBootSmmPageProtection - Enter\n"));
```

Comment:
This function seems sharable between smm and standalone, any reason why they each have their own implementation?

Reply:
Consolidated.